### DISCUSSION TITLE
[Translation] - Alternate Russian translation

### DISCUSSION THREAD

**Discussion by Inui-senpai** [Monika-After-Story/MonikaModDev#7825] [general - translations]
It's the same version as the #3761, but with many grammar, stylistic and technical mistakes fixing.

Here's the link on our translation (check "Последняя версия, переведённая на данный момент времени"):
https://wiki.rgsr.ru/Monika_After_Story#Ссылки
Mirror: https://wiki-rgsr.ffsb.ru/Monika_After_Story#Ссылки

**Comment by Naut-ppc** [comment]
ссылка не работает (

**Reply by Naut-ppc** [reply]
а, нет, работает просто не с первого раза открылась
